src/objects/inline_keyboard.rs

```diff
@@ -0,0 +1,39 @@
+use std::option::Option;
+
+#[derive(Serialize, Debug)]
+pub struct Markup {
+	pub inline_keyboard: Vec<Vec<Button>>,
+}
+
+#[derive(Serialize, Debug)]
+pub struct Button {
+	text: String,
+	url: Option<String>,
+	callback_data: Option<String>,
+	switch_inline_query: Option<String>,
+	switch_inline_query_current_chat: Option<String>,
+}
+
+impl Markup {
```

Comment:
The name of this in the Telegram bot API is "InlineKeyboardMarkup", please try to keep the names as close as possible.